plugins/NitroSpoof/src/main/java/com/xinto/aliuplugins/nitrospoof/PluginSettings.kt

```diff
@@ -37,7 +40,19 @@ class PluginSettings(
             }
         }
 
+        val emptyToggle = Utils.createCheckedSetting(context, CheckedSetting.ViewType.SWITCH, "Enable Empty Character", warning)
+        emptyToggle.isChecked = settingsAPI.getBool("emptyChar", false)
+        emptyToggle.setOnCheckedListener{
+        	c: Boolean? -> 
```

Comment:
the arguments should be placed on the same line as the lambda begins, also add spaces before brackets and parenthesis please